extensions/amp-next-page/1.0/service.js

```diff
@@ -743,9 +757,42 @@ export class NextPageService {
    * @private
    */
   buildDefaultSeparator_() {
-    const separator = this.win_.document.createElement('div');
-    separator.classList.add('amp-next-page-default-separator');
-    return separator;
+    const html = htmlFor(this.getHostNextPageElement_());
```

Comment:
What is this line for?

Reply:
This is so that the template tag function (`html`) becomes available in the next line (`` return html`... ``)